CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## Master (Unreleased)
 
+## 2.27.0 (2024-03-01)
+
 - Add new `RSpec/IsExpectedSpecify` cop. ([@ydah])
+- Add new `RSpec/RepeatedSubjectCall` cop. ([@drcapulet])
 - Add support for `assert_true` and `assert_false` to `RSpec/Rails/MinitestAssertions`. ([@ydah])
-- Support asserts with messages in `Rspec/BeEmpty`. ([@G-Rath])
 - Add support for `assert_empty`, `assert_not_empty` and `refute_empty` to `RSpec/Rails/MinitestAssertions`. ([@ydah])
 - Support correcting some `*_predicate` assertions in `RSpec/Rails/MinitestAssertions`. ([@G-Rath])
 - Support correcting `*_kind_of` assertions in `RSpec/Rails/MinitestAssertions`. ([@G-Rath])
```

Comment:
I was thinking maybe we should squash all these lines about MinitestAssertion.

Reply:
Indeed it is, I updated CHANGELOG.md